### DISCUSSION TITLE
Connection doesn't recognize password database strings

### DISCUSSION THREAD

**Discussion by CoeqWEty** [sidorares/node-mysql2#2832] [Q&A]
I'm new hope you can help me .

 I'm trying to get a connection but the elements of the connection string think they are unknown.  They are literal strings.  see png file What am I missing TIA 

![sql connection](https://github.com/sidorares/node-mysql2/assets/9252209/e2381833-9716-443a-a78b-cab0c274c00e)

**Comment by sidorares** [comment]
Are you sure these are type errors?

**Reply by CoeqWEty** [reply]
It's odd I can type in strings like 'test' for database and '123456' for password, it doesn't display these issues 

It's just standard mysql connection.  I don't think they are type errors, I just don't know why it's reaching in and stating 1 string is good and another bad. The local mysql connection has credentials it says are unknown words. 

Does something else need to be configed  any Ideas?

**Comment by wellwelwel** [comment]
@CoeqWEty, it looks like a _spell checker_ plugin, could you check it? 🙋🏻‍♂️

**Reply by CoeqWEty** [reply]
yes, this is what I found too. You are correct. Im learning and they don't set cspell settings in their tutorials. Anyone learning should make sure cspell spelling in vs code aren't plaguing them because they arent configured in tutorials. It by default reaches inside of strings to spell check as well as a host of other expressions like regex functions, function names, etc.   You can manipulate many settings. https://cspell.org/configuration/document-settings/
